src/System.Threading/tests/SemaphoreSlimTests.cs

```diff
@@ -160,27 +160,6 @@ public static void RunSemaphoreSlimTest7_AvailableWaitHandle()
             RunSemaphoreSlimTest7_AvailableWaitHandle(0, 10, SemaphoreSlimActions.Release, true);
         }
 
-        [Fact]
-        [OuterLoop]
```

Comment:
They were Outerloop I'm assuming because of the values that were used.  I split this up to help identify where the bottlenecks were, and then I changed some of the numbers to eliminate them, at which point it seemed fine to leave it innerloop.  In fact, we should consider just removing [Outerloop] entirely at this point from both this and the System.Threading.Tasks.Parallel tests, since all tests now run sub 10s.